setup.py

```diff
@@ -17,7 +17,7 @@
 
 # Create a local copy for the gin test configuration file based on the master file `base_gin_test_config.json`
 gin_config_file_base = "./base_gin_test_config.json"
-gin_config_file_local = "./tests/test_on_data/gin_test_config.json"
+gin_config_file_local = "gin_test_config.json"
 copy(src=gin_config_file_base, dst=gin_config_file_local)
```

Comment:
I guess you moved this to the top folder for ease of access after moving the config python file to tests, corrrect? or is there other reason that I am missing for this?

My view, as in the last comment, is that all of this should be on `/tests`.

Reply:
Fine to move this into `tests` if you can find a way to get it to work